crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -616,34 +616,38 @@ pub enum RpcModuleSelection {
     #[default]
     Standard,
     /// Only use the configured modules.
-    Selection(Vec<RethRpcModule>),
+    Selection(HashSet<RethRpcModule>),
 }
 
 // === impl RpcModuleSelection ===
 
+/// The standard modules to instantiate by default `eth`, `net`, `web3`
+// TODO: replace with LazyLock
+static STANDARD_SELECTION: OnceLock<HashSet<RethRpcModule>> = OnceLock::new();
+
+/// All modules.
+// TODO: replace with LazyLock
+static ALL_SELECTION: OnceLock<HashSet<RethRpcModule>> = OnceLock::new();
```

Comment:
don't think we need a static for this.

this is only used once --all or for help messages so allocating is better than keeping them around

Reply:
kk. this means removing `as_selection`, and extra clones in `are_identical` seems fine tho